freeipa/src/main/java/com/sequenceiq/freeipa/flow/stack/termination/action/TerminationService.java

```diff
@@ -70,6 +70,7 @@ public void finalizeTermination(Long stackId) {
             });
         } catch (TransactionService.TransactionExecutionException ex) {
             LOGGER.info("Failed to terminate cluster infrastructure. Stack id {}", stack.getId());
+            stackUpdater.updateStackStatus(stackId, DetailedStackStatus.DELETE_FAILED, "Finalization of stack termination failed.");
```

Comment:
Maybe we could also take into account the forced flag.
I guess we want the forced delete functionality to finalize the deletion at any cost.
`if (forced) -> DELETE_COMPLETED`